bin/cfg/Tomb1Main.json5

```diff
@@ -93,17 +93,17 @@
     "healthbar_color": "red",
 
     // Changes how the airbar is displayed. Possible values:
-    // - always: always show the airbar
-    // - default: show the airbar only in the water
-    // - flashing-or-default: show the airbar when Lara's oxygen is 20% or
-    //   below or under normal circumstances
-    // - flashing-only: show the airbar *only* when Lara's oxygen is 20% or /
+    // - default: show the airbar only in the water (as in OG)
+    // - flashing-or-default: flash the airbar when Lara's oxygen is 20% or
+    //   below, or show under normal circumstances
+    // - flashing-only: show and the airbar *only* when Lara's oxygen is 20% or /
     //   below (useful for challenge runs)
+    // - always: always show the airbar
     // - never: never display the airbar (for challenge runs)
     // In case of airbar, the flashing-or-default mode is there mostly for
     // consistency and the only practical difference is that it's shown when
     // Lara dies. When using the fly cheat, the airbar is never shown.
-    "airbar_showing_mode": "default",
+    "airbar_showing_mode": "flashing-or-default",
```

Comment:
This mode does not make much sense TBH for the airbar. You'll never get a flashing airbar outside of a water body.

Reply:
I didn't add this mode. Just rearranged the order to match the order of the json, enums, and case statement. Want me to remove it?

Reply:
I'm not sure, do you think it makes sense to have it around?
When I added it I only made it to be consistent, but I think that might have been a mistake.

Reply:
I don't see a use for it, but I get your point about consistency. It's not harmful other than lengthening the list of options.